core/src/banking_stage/transaction_scheduler/receive_and_buffer.rs

```diff
@@ -421,7 +421,7 @@ impl TransactionViewReceiveAndBuffer {
                     .zip(transaction_priority_ids.iter())
                 {
                     if result.is_err() {
-                        num_dropped_on_status_age_checks += 1;
+                        num_dropped_on_transaction_checks += 1;
```

Comment:
To double check: `result` here is from `working_bank.check_transactions()`, which should be counted at `num_dropped_on_age_and_status` as scheduler_controller does it [here](https://github.com/anza-xyz/agave/blob/master/core/src/banking_stage/transaction_scheduler/scheduler_controller.rs#L261-L271), is it?

Reply:
`check_results` returned at ln403 above seems just the results of `working_bank.check_transactions()`, does it also do fee payer check?

Reply:
Yes, that fee payer is checked later; but at this point, any error is "status and age" error, the count increment should apply to `num_dropped_on_status_age_checks`, no? Or unless the intention is to get ride of `num_dropped_on_status_age_checks`, and count all err (status, age and fee payer) into `num_dropped_on_transaction_checks`?

Reply:
`dropped_on_status_and_age_checks` is only counting status/age during `clean`.
`dropped_on_transaction_checks` is counting status/age + fee-payer checks during `receive_and_buffer`.

The names are not good. Maybe it's best to rename `dropped_on_status_and_age_checks` to `dropped_on_clean` or similar. Thoughts on rename?

Reply:
local `dropped_on_transaction_checks` here is eventually reported as "num_dropped_on_receive_transaction_checks" btw

Reply:
yea, I was confused by the names. Thought the intention was to report "status-age" error to its namesake counter, and such. These counter names are mouthful ;) But practically myself often go back to code, and trust comments for the counters.